packages/frontend/src/components/ApolloProvider/index.tsx

```diff
@@ -1,20 +1,26 @@
 import * as React from 'react'
 import { ApolloProvider as BaseApolloProvider } from '@apollo/client'
+import { useToast } from '@opengovsg/design-system-react'
 import { createClient } from 'graphql/client'
-import { SnackbarMessage, useSnackbar } from 'notistack'
```

Comment:
There's only usage of this left in:
- SGID login component
- AppConnectionRow component
Do you wanna replace in those as well then we can remove this dep entirely?

Reply:
ok lemme go fix this